docs/features/geomodel.md

```diff
@@ -78,8 +78,8 @@ entity borders; four databases are available. These databases are empty by defau
 filled as soon as they are used.
 
  * The `RINGMesh::GeoModelMeshVertices` 
- * The `RINGMesh::GeoModelMeshEdges` this particular database is usfull to represent wells.
- * The `RINGMesh::GeoModelMeshFacets` 
+ * The `RINGMesh::GeoModelMeshEdges` this particular database is useful to represent wells.
```

Comment:
This is wrong since now we have GeoModelMeshEdges for line mesh elements and GeoModelMeshWells for well mesh elements